Dokan/Dokan/Scenes/ProductDetails/ReviewsView.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  ReviewsView.swift
+//  Dokan
+//
+//  Created by Clara Mounir Adly on 16/07/2022.
+//
+import CoreMIDI
+import Foundation
+import UIKit
+
+public class ReviewsView: UIView, UITableViewDelegate, UITableViewDataSource {
+
+    var uiMODEL: ReviewsUiModel?
+
+    // MARK: outlets
+
+    @IBOutlet weak var contentView: UIView!
+    @IBOutlet weak var reviews: UILabel!
+    @IBOutlet weak var rating: UILabel!
+    @IBOutlet weak var customerReview: UITableView!
+    @IBOutlet weak var seeAllReviewsButton: UIButton!
+    @IBOutlet weak var imageReview: UIImageView!
+
+    // MARK: functions
+
+    public func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        uiMODEL!.reviews.count
+    }
+
+    public func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "ReviewerTableViewCell") as! ReviewerTableViewCell
+        guard let data = (uiMODEL?.reviews[indexPath.row]) else { return UITableViewCell() }
+        cell.configureCell(reviewTableViewCellViewModel: data)
+        return cell
+    }
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        commonInit()
+        customerReview.dataSource = self
+        customerReview.delegate = self
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        commonInit()
+    }
+
+    func commonInit() {
+        Bundle.main.loadNibNamed("ReviewsView", owner: self, options: nil)
+        contentView.fixInView(self)
+    }
+}
```

Comment:
The code won't compile due to the need to link the new property names to the nib.. please make each change locally then request review. 
```suggestion
// MARK: - ReviewsView

final class ReviewsView: UIView {

    // MARK: Outlets

    @IBOutlet private(set) weak var contentView: UIView!
    @IBOutlet private(set) weak var reviewsLabel: UILabel!
    @IBOutlet private(set) weak var ratingLabel: UILabel!
    @IBOutlet private(set) weak var tableView: UITableView!
    @IBOutlet private(set) weak var seeAllReviewsButton: UIButton!
    @IBOutlet private(set) weak var reviewImageView: UIImageView!

    // MARK: Properties

    private var viewModel: ReviewsUiModel?

    // MARK: Lifecycle

    override init(frame: CGRect) {
        super.init(frame: frame)
        commonInit()
    }

    required init?(coder aDecoder: NSCoder) {
        super.init(coder: aDecoder)
        commonInit()
    }

    private func commonInit() {
        Bundle.main.loadNibNamed("ReviewsView", owner: self, options: nil)
        contentView.fixInView(self)
        tableView.dataSource = self
        tableView.delegate = self
        tableView.register(ReviewerTableViewCell.self, forCellReuseIdentifier: "\(ReviewerTableViewCell.self)")
    }
}

// MARK: - UITableViewDataSource Conformance

extension ReviewsView: UITableViewDataSource {

    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
        return viewModel?.reviews.count ?? .zero
    }

    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
        guard let cell = tableView.dequeueReusableCell(withIdentifier: "\(ReviewerTableViewCell.self)") as? ReviewerTableViewCell,
              let viewModel = viewModel else {
            return UITableViewCell()
        }

        let review = viewModel.reviews[indexPath.row]
        cell.configureCell(viewModel: review)
        return cell
    }
}

// MARK: - UITableViewDelegate Conformance

extension ReviewsView: UITableViewDelegate {}
```